api/HTMLMediaElement.json

```diff
@@ -300,10 +300,10 @@
               "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": "6"
+              "version_added": "3.2"
             },
             "safari_ios": {
-              "version_added": true
+              "version_added": "3"
```

Comment:
Media playback is a very platform specific thing, so I don't trust mirroring from Safari macOS when it comes to `HTMLMediaElement`, especially its initial launch.

https://caniuse.com/video claims it's since Safari iOS 3.2, but I tried http://mdn-bcd-collector.appspot.com/tests/api/HTMLMediaElement on the oldest iOS simulator BrowserStack has, which is iOS 3.0, and the interfaces are there. The WebKit version in the UA string was 528.18. So this seems correct!